include/biscuit/literal.hpp

```diff
@@ -0,0 +1,175 @@
+#pragma once
+
+#include <biscuit/assert.hpp>
+
+#include <cstddef>
+#include <optional>
+#include <set>
```

Comment:
This should also include `<type_traits>` since `std::is_trivially_copyable` is used in a static assert.

Reply:
Done!